index.html

```diff
@@ -75,8 +88,9 @@ <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
       </div>
     </section>
 
```

Comment:
- [ ] When I click on the hamburger button the mobile menu does not appear. Please kindly fix that. 
- [ ] The background image of the header does not cover up the whole header section on mobile view. Please kindly fix that.

Reply:
Hi Emmanuelle,
Actually, I am surprised about your comment, because I pulled the GitHub codes again and the hamburger menu and cancel (cross) icon work fine on my screen. Can you check it again?

Besides, I didn't understand your comment about the background image. It is almost the same as Figma's design. 

![Screen Shot 2023-01-03 at 15 04 00](https://user-images.githubusercontent.com/44959066/210353946-10684f8a-8b24-42ac-9eeb-8d101aa72e08.png)

Reply:
![one](https://user-images.githubusercontent.com/61285247/210359185-cf3994e0-c2d3-44c3-8ba3-ab50887ebded.png)
This is what am talking about. There should be no space there.

Reply:
It happens due to resizing. I fixed it now though.